docs/tidb_http_api.md

```diff
@@ -56,3 +56,32 @@ curl http://{TiDBIP}:10080/schema/{db}/{table}
 curl http://{TiDBIP}:10080/tables/{db}/{table}/disk-usage
 ```
 
+> 10. Get MVCC info of the key with a specified handle ID
+
+```
+curl http://{TiDBIP}:10080/mvcc/key/{db}/{table}/{handle}
+```
+
+> 11. Get MVCC info of the first key in the table with a specified start ts
+
+```
+curl http://{TiDBIP}:10080/mvcc/txn/{startTS}/{db}/{table}
+```
+
+> 12. Get MVCC info of the key with a specified handle ID
+
+```
+curl http://{TiDBIP}:10080/mvcc/txn/{startTS}
+```
+
+> 13. Get MVCC Information by a hex value
+
+```
+curl http://{TiDBIP}:10080/mvcc/hex/{hexKey}
+```
+
+> 14. Get MVCC Information of index record key
+
+```
+curl http://{TiDBIP}:10080/mvcc/index/{db}/{table}/{index}/{handle}
```

Comment:
Please add these `?${c1}={v1}&${c2}=${v2}` after handle, where c1 and c2 mean column name, and v1 and v2 represent column value.

Reply:
Sorry for my inaccurate comment. I mean 14 should be 15. There is no api looks like http://{TiDBIP}:10080/mvcc/index/{db}/{table}/{index}/{handle}